### PR TITLE
Add OCP 4.7 config file for BM

### PR DESCRIPTION
**Pull request by prsurve**
Signed-off-by: Pratik Surve <prsurve@redhat.com>

### CODE REVIEW

**Review by ebenahar — APPROVED** [review]

File: ocs_ci/deployment/baremetal.py [hunk 1]
```diff
@@ -223,23 +224,24 @@ def deploy_prereq(self):
...
-                assert self.helper_node_handler.exec_cmd(
-                    cmd=cmd
-                ), "Failed to Download required File"
-            else:
-                raise RhcosImageNotFound
+                if check_for_rhcos_images(metal_image_path):
+                    cmd = (
+                        "wget -O "
+                        f"{self.helper_node_details['bm_path_to_upload']}"
+                        "/rhcos-metal.x86_64.raw.gz "
```
Comment (vasukulkarni) [inline ocs_ci/deployment/baremetal.py:235]: can you make this constant at the top of this same file ?

File: ocs_ci/deployment/baremetal.py [hunk 2]
```diff
@@ -498,7 +501,11 @@ def create_pxe_files(self, ocp_version, role):
             extra_data = ""
             bm_install_files_loc = self.helper_node_details["bm_install_files"]
             extra_data_pxe = "rhcos-live-rootfs.x86_64.img coreos.inst.insecure"
-            if ocp_version == "4.6":
+            if ocp_version <= 4.6:
+                bm_metal_loc = f"coreos.inst.image_url={bm_install_files_loc}rhcos-metal.x86_64.raw.gz"
```
Comment (vasukulkarni) [inline ocs_ci/deployment/baremetal.py:505]: can use above constant

File: ocs_ci/deployment/baremetal.py (hunk 1 above)
Comment (prsurve) [reply ocs_ci/deployment/baremetal.py]: Done

File: ocs_ci/deployment/baremetal.py (hunk 2 above)
Comment (prsurve) [reply ocs_ci/deployment/baremetal.py]: Done

**Review by vasukulkarni — APPROVED** [review]

**Comment by prsurve** [comment]
> PR validation
> 
> Cluster Name:
> Cluster Configuration: conf/ocsci/skip_ocs_deploy.yaml
> PR Test Suite: deployment
> PR Test Path: tests/
> Additional Test Params:
> OCP VERSION: 4.7
> OCS VERSION: 4.7
> 
> [Job FAILED](https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-test-pr/3795//testReport) (installation failed, tests not executed).

OCP install passed job failed for timeout for chonyd Machineconfig

**Review by vavuthu — APPROVED** [review]

**Pull request merged**